src/Components/ElementsPanel/Internal/VisualRuleElementsList.tsx

```diff
@@ -0,0 +1,274 @@
+        const elementItemWithColorings: ICardboardGroupedListItem<ITwinToObjectMapping> = {
+            id: rowId,
+            ariaLabel: element.displayName,
+            buttonProps: {
+                customStyles: buttonStyles.elementButton,
+                ...(onItemHover && {
+                    onMouseEnter: () => onItemHover(element, panelItem),
+                    onFocus: () => onItemHover(element, panelItem)
+                }),
+                ...(onItemBlur && {
+                    onMouseLeave: () => onItemBlur(element, panelItem),
+                    onBlur: () => onItemBlur(element, panelItem)
+                })
+            },
+            iconStart: () => (
+                <ElementColoring
+                    rowId={rowId}
+                    colorings={elementColorings}
+                    isModal={isModal}
+                />
+            ),
+            item: element,
+            itemType: 'header',
+            onClick: () => onItemClick(element, panelItem),
+            textPrimary: element.displayName
+        };
+        listItems.push(elementItemWithColorings);
+
+        badges.map((badge) => {
+            const badgeStyles = getElementsPanelAlertStyles(badge.visual.color);
+            const onEnter =
+                onItemHover && (() => onItemHover(element, panelItem));
+            const onLeave =
+                onItemBlur && (() => onItemBlur(element, panelItem));
+            const alertItem: ICardboardGroupedListItem<IExpressionRangeVisual> = {
+                ariaLabel: badge.visualRuleDisplayTitle,
+                buttonProps: {
+                    customStyles: buttonStyles.alertButton,
+                    onMouseEnter: onEnter,
+                    onFocus: onEnter,
+                    onMouseLeave: onLeave,
+                    onBlur: onLeave
+                },
+                iconStart: () => (
+                    <span className={badgeStyles.alertCircle}>
+                        <Icon iconName={badge.visual.iconName} />
+                    </span>
+                ),
+                item: badge.visualRule,
+                itemType: 'item',
+                onClick: () =>
+                    onItemClick(badge.visualRule, panelItem, badge.behavior),
+                textPrimary: badge.visualRuleDisplayTitle
+            };
+            listItems.push(alertItem);
+        });
+    });
+    return listItems;
+}
+
+interface IElementColoringProps {
+    colorings: VisualColorings[];
+    isModal: boolean;
+    rowId: string;
+}
+
+const ElementColoring: React.FC<IElementColoringProps> = (props) => {
+    const tooltipId = useId('cb-element-coloring-header-tooltip');
+    const theme = useExtendedTheme();
+
+    if (!props.isModal) {
+        const tooltipContent = (colorings: VisualColorings[]) => {
+            return <ColorPillsTooltip visualColorings={colorings} />;
+        };
+
+        return (
+            <TooltipHost
```

Comment:
I would argue that the tooltip doesn't make sense here. I would bind this to a callout and show/hide based on the callbacks from CardboardList. The tooltip interaction here feels really weird given the location of the tooltip (showing up 300px away from where I hovered feels wrong). 
I also would expect the whole row to be the target instead of just the bars since hovering the row already has the side effect of highlighting the element in the viewer.